frontend_app_folder_advanced/src/container/FolderAdvanced.jsx

```diff
@@ -121,7 +121,7 @@ class FolderAdvanced extends React.Component {
 
   handleClickBtnCloseApp = () => {
     this.setState({ isVisible: false })
-    GLOBAL_dispatchEvent({type: 'appClosed', data: {}}) // handled by tracim_front::src/container/WorkspaceContent.jsx
+    GLOBAL_dispatchEvent({type: CUSTOM_EVENT.APP_CLOSED, data: {}}) // handled by tracim_front::src/container/WorkspaceContent.jsx
```

Comment:
now we have a variable, it is easier to find where are the usages of a given custom event.
So you can remove this kind of comments